graphics/squish/Portfile

```diff
@@ -19,15 +20,26 @@ description         DXT Compression Library
 
 long_description    DXT compression is a very well-designed compression scheme \
                     for colour textures with an optional alpha channel. The \
-                    squish library is a cross-platform open source \
+                    ${name} library is a cross-platform open source \
                     implementation of DXT compression (and decompression).
 
-homepage            https://sourceforge.net/projects/libsquish/
-master_sites        sourceforge:project/libsquish
-distname            libsquish-${version}
+homepage            https://sourceforge.net/projects/lib${name}/
+master_sites        sourceforge:project/lib${name}
+distname            lib${name}-${version}
 extract.suffix      .tgz
 extract.mkdir       yes
 
+post-patch {
+    reinplace "/^INSTALL/i\\
+install(\\
+\\    FILES lib${name}.pc.in\\
+\\    DESTINATION \${CMAKE_INSTALL_PREFIX}/share/pkgconfig\\
+\\    RENAME lib${name}.pc\\
+\\    )\\
+" \
+        ${worksrcpath}/CMakeLists.txt
```

Comment:
This is just adding these lines to the CMakeLists.txt, yes? This should be a patchfile, not a reinplace.

Reply:
Done.